### PR TITLE
Made a Norwegian translation

### PR DESCRIPTION
**Pull request by DandelionSprout**
I decided to just as well make a Norwegian translation of The Homebrew Channel while I needed something to do this afternoon.

Yes, I know, I know, it's most probably not easy to incorporate a translation for a language that isn't among Nintendo's official system languages. But I figured that I could give it a shot nevertheless.

### CODE REVIEW

File: source/language.c [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #define STR_PT(_str) [CFG_LANGUAGE_PT] = _str
 #define STR_RU(_str) [CFG_LANGUAGE_RU] = _str
 #define STR_TW(_str) [CFG_LANGUAGE_TW] = _str
+#define STR_NO(_str) [CFG_LANGUAGE_NO] = _str
```
Comment (mtheall) [inline source/language.c:15]: I'm assuming `CFG_LANGUAGE_NO` doesn't exist.

Comment (DandelionSprout) [reply source/language.c:15]: That is correct. I wasn't sure what else I could've put in there instead. 😅

Comment (fincs) [reply source/language.c:15]: It doesn't, and therefore the code doesn't compile. Unfortunately we cannot accept languages which don't already exist in Nintendo's language list.

**Comment by fincs** [comment]
I'm sorry, but since Norwegian isn't a language supported by Nintendo's official interface and language detection routines, we cannot accept this pull request.

**Comment by DandelionSprout** [comment]
Okay. Fair enough. 🙂

**Pull request closed without merging**